### ISSUE TITLE
Scrollable tab as content

### ISSUE THREAD

**Issue by mschipperheyn** [jaysoo/react-native-parallax-scroll-view#39]
Hi,

Really like the component! I'm trying to implement a sticky header that simply contains the navigation bar, which works fine. However, the scrollable content contains ScrollableTabView (react-native-scrollable-tab-view. I have two problems that perhaps you may have a suggestion for:
1. The scrollable tabs need to stick the top. If the page is longer than the screen, they basically scroll out of view
2. I need to scroll on the parallax ScrollView when it's still visible, but transparently switch to the scrollview in the tab when it is not. Now, the scrolling depends on where you put your finger. I tried locking scrolling on the child ScrollViews, also, to try prevent scrolling the ScrollableTabView out of sight, but this didn't work.

Any suggestions?

**Comment by viix** [comment]
@mschipperheyn , same requirements, have you got solution?

**Comment by mschipperheyn** [comment]
No. My idea was to process scroll events but our dev team is not succeeding. I took them off this.

**Comment by viix** [comment]
OK, was thinking the same approach.
any way, thanks for your info.

**Comment by mschipperheyn** [comment]
Let me know if u come up w something. We simply didn't have the time

**Comment by infacq** [comment]
I also want to combine it with `tabBarIos` but no success

**Comment by mschipperheyn** [comment]
It would be nice to be able to detect when the parallax animation has completed its open and close cycle. With that we can enable / disable scrolling on child listviews so this problem doesn't occur.

**Comment by tanpuer** [comment]
I really like this component . But it is not compatible with react-native-scrollable-tab-view.
If you have fixed this problem, can you share with me?

**Comment by tanpuer** [comment]
Facebook f8 app provides ListContainer, it is similar with the combination of ScrollableTabView and ParallaxView. I have a try, it does work!

**Comment by boniattirodrigo** [comment]
Do you have some solutions for this case? I've searched for parallax with tabs, but at the moment I found nothing yet.

**Comment by shukerullah** [comment]
Same Issue is this me, I am looking for solution.

**Comment by kimooc** [comment]
解决了吗

**Comment by h1nson** [comment]
i need a clone of instagram profile screen
